.circleci/config.yml

```diff
@@ -58,7 +58,7 @@ jobs:
           path: /tmp/test-results
       - run:
           name: Publish pacts
-          command: PACT_TAG=$CIRCLE_BRANCH PACT_CONSUMER_VERSION=$(echo $CIRCLE_SHA1 | cut -c -7) go run internal/pact/publish.go
+          command: PACT_TAG=$CIRCLE_BRANCH PACT_CONSUMER_VERSION=$(echo $CIRCLE_SHA1) go run internal/pact/publish.go
```

Comment:
do you need the echo now, `PACT_CONSUMER_VERSION=$CIRCLE_SHA1`?

Reply:
Oh yeah good point 😅